### PR TITLE
small typo fixed in 04-class-field.tex

### PR DESCRIPTION
**Pull request by iamrajiv**

### CODE REVIEW

File: HaxeManual/04-class-field.tex [hunk 1]
```diff
@@ -195,7 +195,7 @@ \section{Method}
 \subsection{Overriding Methods}
 \label{class-field-overriding}
 
-Overriding fields is instrumental for creating class hierarchies. Many design patterns utilize it, but here we will explore only the basic functionality. In order to use overrides in a class, it is required that this class has a \tref{parent class}{types-class-inheritance}. Let us consider the following example:
+Overriding fields are instrumental for creating class hierarchies. Many design patterns utilize it, but here we will explore only the basic functionality. In order to use overrides in a class, it is required that this class has a \tref{parent class}{types-class-inheritance}. Let us consider the following example:
```
Comment (Gama11) [inline HaxeManual/04-class-field.tex]: That was correct before.

File: HaxeManual/04-class-field.tex [hunk 2]
```diff
@@ -264,7 +264,7 @@ \subsection{Visibility}
 \subsection{Inline}
 \label{class-field-inline}
 
-The \expr{inline} keyword allows function bodies to be directly inserted in place of calls to them. This can be a powerful optimization tool, but should be used judiciously as not all functions are good candidates for inline behavior. The following example demonstrates the basic usage:
+The \expr{inline} keyword allows function bodies to be directly inserted in the place of calls to them. This can be a powerful optimization tool but should be used judiciously as not all functions are good candidates for inline behavior. The following example demonstrates the basic usage:
```
Comment (Gama11) [inline HaxeManual/04-class-field.tex]: This also sounded better before.

File: HaxeManual/04-class-field.tex [hunk 3]
```diff
@@ -140,7 +140,7 @@ \subsection{Rules for getter and setter}
 
 Visibility of accessor methods has no effect on the accessibility of its property. That is, if a property is \expr{public} and defined to have a getter, that getter may be defined as \expr{private} regardless.
 
-Both getter and setter may access their physical field for data storage. The compiler ensures that this kind of field access does not go through the accessor method if made from within the accessor method itself, thus avoiding infinite recursion:
+Both getter and setter may access their physical field for data storage. The compiler ensures that this kind of field access does not go through the accessor method is made from within the accessor method itself, thus avoiding infinite recursion:
```
Comment (Aurel300) [inline HaxeManual/04-class-field.tex]: No, it's saying the physical access is used *when* made from within the accessor method. It was correct as it was.

**Comment by Aurel300** [comment]
Thanks!

**Pull request merged**